src/containers/RapidAppointment/RapidAppointment.tsx

```diff
@@ -117,6 +118,16 @@ export function RapidAppointment() {
     }
   }, [locationData, previousData]);
 
+  useEffect(() => {
+    if (isCreateRequestSuccessful) {
```

Comment:
Generally at the end of a flow I would `return <Redirect...` as I *think* it's faster than `useEffect` since `useEffect` enqueues a message in the event loop, but it's not that big of a deal and the way `history` is setup in our tests makes it so convenient to test for path and search params.